app/src/renderer/vuex/modules/delegates.js

```diff
@@ -70,10 +50,10 @@ export default ({ node }) => {
           validator.pub_key
         )
         if (!isEmpty(signing_info)) validator.signing_info = signing_info
-        commit(`addDelegate`, validator)
       }
+      commit(`setDelegates`, validators)
     },
-    async getDelegates({ state, commit, dispatch }) {
+    async getDelegates({ commit, dispatch }) {
```

Comment:
`getValidators`

Reply:
I won't do API changes in this PR as it would increase the complexity of this PR immensely.